transactions/WFLY-14869-microprofile-lra.adoc

```diff
@@ -0,0 +1,128 @@
+= Support for MicroProfile LRA
+:author:            Martin Stefanko
+:email:             mstefank@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+:keywords:          transaction, saga, lra, microprofile
+
+== Overview
+
+The goal of this RFE is providing the Galleon feature pack for WildFly which integrates the
+https://github.com/jbosstm/narayana/tree/master/rts/lra[Narayana implementation] of the
+https://github.com/eclipse/microprofile-lra[MicroProfile LRA specification].
+
+The MicroProfile LRA is a transaction processing pattern well suited for microservice development based on the saga
+pattern. The motivation of such processing is summarized in the
+https://github.com/eclipse/microprofile-lra/blob/1.0/README.adoc#motivation[MicroProfile Spec readme].
+
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.redhat.com/browse/WFLY-14869
+
+=== Related Issues
+
+* https://issues.redhat.com/browse/EAP7-1496
```

Comment:
I am not sure it makes sense to have a product tracker here in the community proposal, since it is not public. WDYT?

Reply:
It seems this is aligned with other proposals. This is also for our reference.

Reply:
Right then, let's leave it there :slightly_smiling_face:

Reply:
👍